packages/gestalt/src/ActivationCard.css

```diff
@@ -1,11 +1,11 @@
 html[dir="rtl"] .rtlPos {
-  left: 0;
+  left: 16px;
   position: absolute;
-  top: 0;
+  top: 16px;
 }
 
 html:not([dir="rtl"]) .rtlPos {
   position: absolute;
-  right: 0;
-  top: 0;
-}
+  right: 16px;
```

Comment:
context for this change?

Reply:
Context: Padding was previously applied in JS via <IconButton padding={4}/>, but this component is now replaced with InternalDismissButton which does not have styling props, so the padding is applied via CSS to its parent, rtl class.

Reply:
##Before
![Screenshot by Dropbox Capture](https://github.com/pinterest/gestalt/assets/10593890/eb9d7b07-9836-4dff-8371-0d593a7717da)

##After 
![Screenshot by Dropbox Capture](https://github.com/pinterest/gestalt/assets/10593890/2e8d5a00-35f6-4227-9603-2a3c58f600ba)

Reply:
@willsmanley Just doublechecking that this was intentional. I think it's ok as it standardizes the sizing of our DismissButtons.